jetty9/pom.xml

```diff
@@ -27,12 +27,6 @@
       <version>${project.version}</version>
       <type>tar.gz</type>
     </dependency>
-    <dependency>
```

Comment:
Perfect... remaining issue is the flag to turn this off when we can

On Tue, Feb 23, 2016 at 11:49 AM, Greg Wilkins <notifications@github.com>
wrote:

> In jetty9/pom.xml
> <https://github.com/GoogleCloudPlatform/appengine-java-vm-runtime/pull/182#discussion_r53838495>
> :
>
> > @@ -27,12 +27,6 @@
> >        <version>${project.version}</version>
> >        <type>tar.gz</type>
> >      </dependency>
> > -    <dependency>
>
> I just tried to hit the site without a Host header and it failed because
> it is a virtual host sharing an IP address. So no host header will always
> result in a failed request.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/appengine-java-vm-runtime/pull/182/files#r53838495>
> .
>